File: Android - Native/discover_destination/app/src/main/java/com/example/discover_destination/screens/StartScreen.kt
package com.example.discover_destination.screens

import androidx.compose.foundation.Image
import androidx.compose.foundation.layout.Column
import androidx.compose.foundation.layout.fillMaxWidth
import androidx.compose.foundation.layout.padding
import androidx.compose.foundation.layout.size
import androidx.compose.material.*
import androidx.compose.runtime.Composable
import androidx.compose.ui.Alignment
import androidx.compose.ui.Modifier
import androidx.compose.ui.graphics.Color
import androidx.compose.ui.unit.dp
import androidx.compose.ui.unit.sp
import coil.compose.rememberImagePainter
import com.example.discover_destination.R

@Composable
fun StartScreen(
    onAddClick: () -> Unit,
    onDeleteClick: () -> Unit,
    onModifyClick: () -> Unit,
    onAllClick: () -> Unit
) {
    Column(
        horizontalAlignment = Alignment.CenterHorizontally,
        modifier = Modifier
            .padding(8.dp)
            .fillMaxWidth()
    ) {
        Image(
            modifier = Modifier.size(400.dp, 400.dp),
            painter = rememberImagePainter(
                R.drawable.logo5
            ),
            contentDescription = null
        )
        OutlinedButton(
            onClick = onAddClick,
            modifier = Modifier.size(300.dp, 70.dp).padding(10.dp)
        ) {
            Text(
                text = "ADD DESTINATION",
                fontSize = 20.sp,
                color = Color.Black)
        }
        OutlinedButton(
            onClick = onDeleteClick,
            modifier = Modifier.size(300.dp, 70.dp).padding(10.dp)
        ) {
            Text(
                text = "DELETE DESTINATION",
                fontSize = 20.sp,
                color = Color.Black)
        }
        OutlinedButton(
            onClick = onModifyClick,
            modifier = Modifier.size(300.dp, 70.dp).padding(10.dp)
        ) {
            Text(
                text = "MODIFY DESTINATION",
                fontSize = 20.sp,
                color = Color.Black)
        }
        OutlinedButton(
            onClick = onAllClick,
            modifier = Modifier.size(300.dp, 70.dp).padding(10.dp)
        ) {
            Text(
                text = "ALL DESTINATIONS",
                fontSize = 20.sp,
                color = Color.Black)
        }
    }
}

File: Flutter - Non-Native/discover_destination/lib/view/start_screen.dart
import 'dart:io';

import 'package:discover_destination/view/add_screen.dart';
import 'package:discover_destination/view/delete_screen.dart';
import 'package:discover_destination/view/modify_screen.dart';
import 'package:discover_destination/view/view_all_screen.dart';
import 'package:discover_destination/viewmodel/destinations_view_model.dart';
import 'package:flutter/material.dart';

class StartScreen extends StatefulWidget {
  DestinationsViewModel destinationsViewModel;
  String user;

  StartScreen(this.destinationsViewModel, this.user, {Key? key}) : super(key: key);

  @override
  _StartState createState() => _StartState(destinationsViewModel, user);
}

class _StartState extends State<StartScreen> {
  DestinationsViewModel destinationsViewModel;
  String user;

  _StartState(this.destinationsViewModel, this.user);

  @override
  Widget build(BuildContext context) {
    return Scaffold(
        backgroundColor: Colors.white,
        body: SingleChildScrollView(
            child: Column(children: <Widget>[
          Padding(
            padding: const EdgeInsets.only(top: 60.0),
            child: Center(
              child: Container(
                  width: 300,
                  height: 300,
                  child: Image.asset("images/logo5.jpg")
              ),
            ),
          ),
          Container(
              decoration: BoxDecoration(
                  color: Colors.white, borderRadius: BorderRadius.circular(20)),
              child: Column(children: <Widget>[
                SizedBox(height: 10),
                OutlinedButton(
                  onPressed: () {
                    Navigator.push(
                        context,
                        MaterialPageRoute(
                            builder: (_) => AddScreen(destinationsViewModel, user)));
                  },
                  child: Text(
                    'Add Destination',
                    style: TextStyle(color: Colors.black, fontSize: 25),
                  ),
                style: OutlinedButton.styleFrom( minimumSize: Size(300, 50)),),
                SizedBox(height: 10),
                OutlinedButton(
                  onPressed: () {
                    Navigator.push(
                        context,
                        MaterialPageRoute(
                            builder: (_) => DeleteScreen(destinationsViewModel, user)));
                  },
                  child: Text(
                    'Delete Destination',
                    style: TextStyle(color: Colors.black, fontSize: 25),),
                  style: OutlinedButton.styleFrom( minimumSize: Size(300, 50)),),
                SizedBox(height: 10),
                OutlinedButton(
                  onPressed: () {
                    Navigator.push(
                        context,
                        MaterialPageRoute(
                            builder: (_) => ModifyScreen(destinationsViewModel, user)));
                  },
                  child: Text(
                    'Modify Destination',
                    style: TextStyle(color: Colors.black, fontSize: 25),
                  ),
                  style: OutlinedButton.styleFrom( minimumSize: Size(300, 50)),),
                SizedBox(height: 10),
                OutlinedButton(
                  onPressed: () {
                    Navigator.push(
                        context,
                        MaterialPageRoute(
                            builder: (_) => ViewAllScreen(destinationsViewModel, user)));
                  },
                  child: Text(
                    'View All Destinations',
                    style: TextStyle(color: Colors.black, fontSize: 25),
                  ),
                 style: OutlinedButton.styleFrom( minimumSize: Size(300, 50)),)
              ]
          )
          )
        ])));
  }
}
